### PR TITLE
Require shinyswatch 0.7.0 and update examples accordingly

### PR DESCRIPTION
**Pull request by cpsievert**
Will fix newly broken playwright tests

### CODE REVIEW

**Review by schloerke — APPROVED** [review]

File: examples/dataframe/app.py [hunk 1]
```diff
@@ -35,6 +34,7 @@ def app_ui(req):
             right="10px",
             bottom="10px",
         ),
+        theme=shinyswatch.theme.darkly() if dark else None,
```
Comment (gadenbuie) [inline examples/dataframe/app.py:37]:
```suggestion
        theme=shinyswatch.theme.darkly if dark else None,
```

File: examples/duckdb/app.py [hunk 2]
```diff
@@ -63,6 +62,7 @@ def load_csv(con, csv_name, table_name):
             ),
         ),
     ),
+    theme=shinyswatch.theme.flatly(),
```
Comment (gadenbuie) [inline examples/duckdb/app.py:65]:
```suggestion
    theme=shinyswatch.theme.flatly,
```

File: shiny/api-examples/todo_list/app-core.py [hunk 3]
```diff
@@ -15,6 +14,7 @@
         ui.output_text("cleared_tasks"),
         ui.div(id="tasks", style="margin-top: 0.5em"),
     ),
+    theme=shinyswatch.theme.minty(),
```
Comment (gadenbuie) [inline shiny/api-examples/todo_list/app-core.py:17]:
```suggestion
    theme=shinyswatch.theme.minty,
```

**Review by gadenbuie — APPROVED** [review]
Looks great, thanks! Now that shinyswatch uses `shiny.ui.Theme`, the callable syntax is historical rather than preferred usage

**Pull request merged**